src/xapiwrapper.js

```diff
@@ -1369,6 +1367,186 @@ function isDate(date) {
         xhr.send(null);
     }
 
+    var isNode = Boolean(!root.document);
+
+    if (isNode) {
+        // Import encoding-indexes as required by text-encoding module
+        var eIndexes = require('../node_modules/text-encoding/lib/encoding-indexes');
```

Comment:
In node, because the `text-encoding/lib/encoding.js` file has been bundled into the `xapiwrapper.min` file it has lost its original relative path. In node the bundled file requires the `encoding-indexes` hash, which is not needed in the browser. These lines import the `encoding-indexes` file into the global scope as they would be if the `encoding.js` file were in its original location. It isn't great but it does work.